### PR TITLE
windows: add tech preview section

### PR DESCRIPTION
**Pull request by ml8mr**
The Windows driver is still being offered as a tech preview. Add
a section to clarify this status.

Signed-off-by: Mike Latimer <mlatimer@suse.com>

### CODE REVIEW

File: xml/windows-ses.xml [hunk 1]
```diff
@@ -54,6 +54,22 @@
      </listitem>
    </itemizedlist>
  </sect1>
+ <sect1 xml:id="technology-preview">
+   <title>Technology Preview</title>
+   <para>
+     &productname; Driver for Windows is currently being offered as a
+     technology preview. This is a necessary step toward full support
+     as we continue work to ensure this driver performs well in all
+     environments and workloads. Please contribute to this effort by
```
Comment (asettle) [inline xml/windows-ses.xml:63]:
```suggestion
     environments and workloads. You can contribute to this effort by

File: xml/windows-ses.xml [hunk 2]
```diff
@@ -54,6 +54,22 @@
...
+     reporting any issues you may encounter to &suse; Support.
+   </para>
+   <para>
+     CephFS functionality requires a third party FUSE wrapper provided
```
Comment (asettle) [inline xml/windows-ses.xml:67]:
```suggestion
     &cephfs; functionality requires a third party FUSE wrapper provided

**Review by asettle — COMMENTED** [review]
Small things.

**Review by asettle — APPROVED** [review]

File: xml/windows-ses.xml [hunk 3]
```diff
@@ -54,6 +54,22 @@
...
+   <para>
+     &productname; Driver for Windows is currently being offered as a
+     technology preview. This is a necessary step toward full support
+     as we continue work to ensure this driver performs well in all
+     environments and workloads. You can contribute to this effort by
+     reporting any issues you may encounter to &suse; Support.
+   </para>
+   <para>
+     &cephfs; functionality requires a third party FUSE wrapper provided
+     through the <link xlink:href="https://github.com/dokan-dev/dokany">dokany project</link>.
```
Comment (lproven) [inline xml/windows-ses.xml:68]: I suggest capitalizing Dokany, as they do upstream.

**Review by lproven — COMMENTED** [review]
Capitalisation change only.

File: xml/windows-ses.xml (hunk 3 above)
Comment (ml8mr) [reply xml/windows-ses.xml:68]: Agreed - https://github.com/SUSE/doc-ses/pull/790

**Pull request merged**